PerfectWedding/src/main/java/com/infoshareacademy/ProvidersEdit.java

```diff
@@ -61,6 +61,7 @@ public void editSelectedProvider(int providerId) {
                     break;
                 case "location":
```

Comment:
Tu jeszcze jedna kwestia - w tekście o edycji mamy "Które pole chciałbyś edytować? (podaj nazwę pola)":
Wyświetlone jest voivodeship jednak w switch nie ma tego osobno więc podanie voivodeship nie wywołuje oczekiwanej przez użytkownika akcji.

Proponuję dodać switch na voivodeship, aby było to zgodne z wyświetlanymi polami w konsoli. 
Kwestia rozwiązania zależy od podejścia do edycji województwo/miasto :) jednak generalnie (aktualnie) działa to tak, że pola w switch są zgodne z tymi wyświetlanymi w princie z konsoli.